core/net.go

```diff
@@ -795,6 +857,9 @@ func (n *OpenBazaarNode) SendOrderPayment(peerID string, paymentMessage *pb.Orde
 	if err != nil {
 		return err
 	}
+
+	n.SendRelayedMessage(p, nil, &m) // send relayed message immediately
```

Comment:
Same question as above:

`SendRelayedMessage` is called in `SendMessage` which is called below. Why call it twice?`

Reply:
Same here, not seeing it.